source/MainWindow.cpp

```diff
@@ -94,6 +76,29 @@ MainWindow::BuildLayout()
 		.Add(fMenuBar)
 		.Add(fTipsterView)
 		.AddGlue();
+
+	// After the creation of Tipster because we need the delay
+	// time saved in the settings
+
+	AddDelaySubMenuItem(30000000, B_TRANSLATE("30 seconds"));
+	AddDelaySubMenuItem(60000000, B_TRANSLATE("1 minute"));
+	AddDelaySubMenuItem(120000000, B_TRANSLATE("2 minutes"));
+	AddDelaySubMenuItem(300000000, B_TRANSLATE("5 minutes"));
+
+	fDelaySubMenu->SetRadioMode(true);
+
+	GetLayout()->SetExplicitMinSize(BSize(740, B_SIZE_UNSET));
```

Comment:
Setting the minimal window width to 740 isn't so nice. I know, making the window too small (so that the tip will linebreak to more than 3 lines) will "scroll" the icon out of sight, but that's so obvious that the user can avoid it.

Of course, ideal would be to  find the longest tip, account for the system font size and limit the min width to 3 lines of that... :)